tools/wptserve/wptserve/handlers.py

```diff
@@ -175,13 +175,14 @@ def load_headers(self, request, path):
             use_sub = False
 
         try:
-            with open(headers_path) as headers_file:
+            with open(headers_path, "rb") as headers_file:
                 data = headers_file.read()
         except IOError:
             return []
         else:
             if use_sub:
                 data = template(request, data, escape_type="none")
+            data = data.decode("ascii")
```

Comment:
Hmm. I suspect we actually want ISO-8859-1 here? We should definitely be able to have non-ASCII in `.headers`. (I presume there's some reason why we need unicode strings here?)

Reply:
Because `template()` works on unicode strings, though maybe that was the wrong call.

Reply:
No, I misread; `template()` works on bytes. `ResponseHeaders` handles both, but its callers expect text.